src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -134,6 +134,16 @@ struct H5IsBondiData {
   using group = Cce;
 };
 
+struct H5SpecNormalization {
```

Comment:
How would you feel about an "active" name like `ApplyH5SpecNormalization` or `FixSpecNormalization` or similar? Perhaps these particular suggestions are less accurate and bad... but my concern is that when reading an input file it's not quite clear to the user if this option is a statement about the input data, the output data, or an action taken in the code. I think using an "active" name for the option will make this clearer. Thoughts?

Reply:
I'll change it to `FixSpecNormalization` -- I do think that's more informative.

Reply:
I'll change it to `FixSpecNormalization` -- I do think that's more informative.